SDK/AppCenterPush/Microsoft.AppCenter.Push.Windows.Shared/Ingestion/Models/PushInstallationLog.cs

```diff
@@ -39,8 +39,10 @@ public class PushInstallationLog : Log
         /// Concrete types like StartSessionLog or PageLog are always part of a
         /// session and always include this identifier.
         /// </param>
-        public PushInstallationLog(System.DateTime? timestamp, Device device, string pushToken, System.Guid? sid = default(System.Guid?))
-            : base(device, timestamp, sid)
+        /// <param name="userId">The user id that user provides in the app.
```

Comment:
@achocron don't we need to regenerate models?

Reply:
Yeah. @DorothySun216 I can help you do that.